appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -292,11 +292,17 @@ Blockly.ReplMgr.pollYail = function(workspace, opt_force) {
 };
 
 Blockly.ReplMgr.resetYail = function(partial) {
-    top.ReplState.phoneState.initialized = false; // so running io stops
+    var rs = top.ReplState;
+    rs.initialized = false; // so running io stops
```

Comment:
Is this supposed to be rs.phoneState.initialized = false ?

Reply:
Fixed